### ISSUE TITLE
Draging a png over Spyder results in an internal problem

### ISSUE THREAD

**Issue by gerazov** [spyder-ide/spyder#8527]
### What steps will reproduce the problem?

Take a file from file browser drag it on top of Spyder gives an error. I didn't mean to drop it in Spyder - I was just moving across to another window.

### Traceback
```python-traceback
  File "/home/vibe/miniconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 2481, in dragEnterEvent
    text = [encoding.is_text_file(url) for url in all_urls]
TypeError: 'NoneType' object is not iterable
```

## Versions

* Spyder version: 4.0.0b1 
* Python version: 3.6.3
* Qt version: 5.6.2
* PyQt5 version: 5.6
* Operating System: Linux 4.19.13-1-MANJARO

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.5 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.11.0     :  0.12.1 (OK)
psutil >=0.3      :  5.4.6 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.14.3 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  2.2.3 (OK)
pylint >=0.25     :  1.9.2 (OK)
```

**Comment by ccordoba12** [comment]
This error is already fixed and it'll be part of beta2 (it's already available in 3.3.2 too).

**Comment by gerazov** [comment]
Ok, great! 
It was popping out 10 times yesterday, I thought might as well send it...
